src/StackAdmin/Azs.Subscriptions.Admin/Module/Azs.Subscriptions.Admin/Azs.Subscriptions.Admin.psd1

```diff
@@ -121,19 +121,27 @@ Licensed under the MIT License. See License.txt in the project root for license
         PSData = @{
 
             # Tags applied to this module. These help with module discovery in online galleries.
-            # Tags = @()
+            Tags         = @('AzureStack', 'Azure Stack', 'Admin', 'Subscriptions', 'Azure Stack Admin Subscription', 'Offer', 'Plan', 'Managed')
 
             # A URL to the license for this module.
-            # LicenseUri = ''
+            LicenseUri   = 'https://aka.ms/azps-license'
 
             # A URL to the main website for this project.
-            # ProjectUri = ''
+            ProjectUri   = 'https://github.com/Azure/azure-powershell'
 
             # A URL to an icon representing this module.
             # IconUri = ''
 
             # ReleaseNotes of this module
-            # ReleaseNotes = ''
+            ReleaseNotes = '## 2018.8.12 - Version 0.2.0
+            * Module dependencies updated
+                * AzureRM.Profile >= 5.3.4
```

Comment:
lets remove the versions from the module release notes? we will keep it on the uber module..